### ISSUE TITLE
Rename "moved" status to "replaced", to make it more generic

### ISSUE THREAD

**Issue by waldyrious** [jantman/repostatus.org#26]
As discussed in #19 (particularly from [this comment](https://github.com/jantman/repostatus.org/issues/19#issuecomment-396016685) onwards), switching from "moved" to "replaced" would support a wider variety of real-world statuses, without significant loss of clarity.

This change would allow the status to cover, for example:
- projects that are renamed within the same platform
- projects that are moved to a different platform
- projects that are superseded by more comprehensive or more actively maintained ones

**Comment by jantman** [comment]
I think this makes sense to me. I don't really have any projects that these new use cases cover, so I'll defer to consensus of interested parties on this. My own needs from RepoStatus are pretty simple, so at this point I'm mainly just managing PRs and community consensus.

Assuming we want to make this switch, I'll have to think about how to handle and communicate a rename of one of the statuses...

**Comment by Alarm-Siren** [comment]
I agree that "Replaced" is a better general-purpose name for the purposes as articulated by @waldyrious 

In terms of managing the switch over, I would suggest just make Moved and Replaced the same thing semantically; Moved is deprecated and Replaced is recommended, but ultimately they mean the same thing.  Its probably unrealistic to get everyone to "upgrade" to Replaced anyway, particularly for older repos that have been long since forgotten.

**Comment by Relequestual** [comment]
Coming here to note that I'd also value "replaced".
In terms of making such an change having as least friction as possible, allowing both seems like a viable solution with the same semantics, and just let the end user choose which they prefer.

I'm happy to make a PR for this if that's helpful, although I probably won't see notifications for some time =]
